File: src/Iniciante/2717.java
package Iniciante;

import java.io.IOException;
import java.util.*;
import java.util.HashMap;
import java.util.Map;
import java.util.regex.*;

class Main {

    public static void main(String[] args) throws IOException {
        Scanner input = new Scanner(System.in);

        int n, a, b;
        n = input.nextInt();
        a = input.nextInt();
        b = input.nextInt();

        if (a + b <= n) {
            System.out.println("Farei hoje!");

        } else {
            System.out.println("Deixa para amanha!");

        }

    }
}

File: src/Iniciante/2717.c
#include <stdio.h>
#include <stdlib.h>
#include <locale.h>
#include <math.h>

int main()
{
      setlocale(LC_ALL, "");
      int n, a, b;
      scanf("%d", &n);
      scanf("%d %d", &a, &b);
      if (a + b <= n)
      {
            printf("Farei hoje!\n");
      }
      else
      {
            printf("Deixa para amanha!\n");
      }

      return 0;
}